pkg/render/logstorage.go

```diff
@@ -1279,6 +1282,19 @@ func (m managedClusterLogStorage) linseedExternalRolesAndBindings() ([]*rbacv1.C
 		},
 	}
 
+	namespacesRole := &rbacv1.ClusterRole{
```

Comment:
We should add a comment explaining why we need this permission, like we do for the other cluster roles in this function.